linker/Linker.Steps/MarkStep.cs

```diff
@@ -1750,6 +1754,25 @@ protected virtual void ProcessMethod (MethodDefinition method)
 		protected virtual void DoAdditionalMethodProcessing (MethodDefinition method)
 		{
 		}
+		
+		void MarkBaseHierarchyAsRequired (TypeDefinition type)
+		{
+			Annotations.MarkBaseRequired (type);
```

Comment:
I'm not ok with additional processing which is unused

Reply:
Ok.